tests/integration/app-charm/actions.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+make-admin:
+  description: Adds admin to relation data
+
+remove-admin:
+  description: Adds admin to relation data
```

Comment:
is this description correct?

Reply:
Yeah should be! It's the 'server-server' auth user, so ideally we'd want it to have super perms across each user.